src/main/java/com/smartling/marketo/sdk/domain/campaign/TriggerCampaignRequest.java

```diff
@@ -0,0 +1,36 @@
+package com.smartling.marketo.sdk.domain.campaign;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+public class TriggerCampaignRequest {
+
+    private LeadId[] leads;
+    private Token[] tokens;
+
+    @Getter
+    @Setter
+    @ToString
+    @AllArgsConstructor
+    public static class LeadId {
```

Comment:
According Marketo https://developers.marketo.com/rest-api/endpoint-reference/lead-database-endpoint-reference/#/Campaigns/triggerCampaignUsingPOST
it should be separate domain class InputLead